python/fledge/plugins/south/person_detection/inference.py

```diff
@@ -63,6 +64,8 @@ def get_interpreter(self, model, enable_tpu, labels, min_conf_threshold=0.5):
                     self.interpreter = Interpreter(model_path=model,
                                                    experimental_delegates=[load_delegate(EDGETPU_SHARED_LIB,
                                                                          {'device': device[0]} if device else {})])
+                    interpreter_loaded = True
```

Comment:
Is it possible "self.interpreter" is not set in above line due to some failure?

Reply:
**interpreter_loaded** will not be set true if exception occurs.